Report.java

```diff
@@ -201,5 +204,61 @@ private void printSpriteCounts(PrintWriter printW, int i)
                 + sprites[j].getSoundBlocksForSprite());
             printW.println();
         }
-    }   
+    }
+
+    /**
+     * Print total variable usage counts.
+     *
+     * @param printW - the printWriter to use
+     * @param i - the submission to get count for
+     */
+    public void printTotalVariableUsage(PrintWriter printW, int i)
```

Comment:
Not to be nitpicky, but can we use a consistent naming convention for these methods? printVariableUsageForProgram, ForStage, ForSprite?